mdast_util_to_markdown/src/configure.rs

```diff
@@ -65,6 +65,10 @@ pub struct Options {
     /// Setext headings cannot be used for empty headings or headings with a
     /// rank of three or more.
     pub setext: bool,
+    /// (`bool`, default: `true`) — whether to support math (text) with a single dollar.
```

Comment:
```suggestion
    /// Whether to support math (text) with a single dollar (`bool`, default: `true`).
```

I place this at the end of the sentence